### PR TITLE
[PC-1180] Portenta X8 Tutorials Banners

### PR DESCRIPTION
**Pull request by canchebagur**
## What This PR Changes
- This PR adds banners to all of the Portenta X8 tutorials.

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/tutorials/11.display-output-webgl/hero-banner.png
Comment (TaddyHC) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/11.display-output-webgl/hero-banner.png:1]: For this, WebGL is the keyword, so maybe something like "WebGL Output" or "Displaying WebGL"?

**Review by TaddyHC — COMMENTED** [review]
I have marked some items if some alternatives fit better for the tutorial's overview. Please have a look at it 🙂

File: content/hardware/04.pro/boards/portenta-x8/tutorials/04.python-arduino-data-exchange/hero-banner.png
[the remark this answers is not in the document]
Comment (TaddyHC) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/04.python-arduino-data-exchange/hero-banner.png:1]: Or simply "Data Exchange"?

File: content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/hero-banner.png
[the remark this answers is not in the document]
Comment (canchebagur) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/hero-banner.png:1]: @jcarolinares what do you think?

File: content/hardware/04.pro/boards/portenta-x8/tutorials/04.python-arduino-data-exchange/hero-banner.png
[the remark this answers is not in the document]
Comment (canchebagur) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/04.python-arduino-data-exchange/hero-banner.png:1]: @jcarolinares what do you think?

File: content/hardware/04.pro/boards/portenta-x8/tutorials/06.waves-fleet-managment/hero-banner.png
[the remark this answers is not in the document]
Comment (canchebagur) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/06.waves-fleet-managment/hero-banner.png:1]: @jcarolinares what do you think?

File: content/hardware/04.pro/boards/portenta-x8/tutorials/11.display-output-webgl/hero-banner.png
Comment (canchebagur) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/11.display-output-webgl/hero-banner.png:1]: @jcarolinares what do you think?

Comment (jcarolinares) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/11.display-output-webgl/hero-banner.png:1]: For this case I agree with @TaddyHC the topic of the tutorial is more related with WebGL Output or displaying WebGL than Video output

Please @canchebagur proceed with one of the @TaddyHC options

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Looks great for me, changes requested regarding the naming of some banners, waiting for that to merge

**Review by jcarolinares — APPROVED** [review]
Thanks for the changes to both

@canchebagur this is ready to merge, please proceed.

**Pull request merged**